src/tools/opt-dist/src/utils/mod.rs

```diff
@@ -70,6 +70,16 @@ pub fn with_log_group<F: FnOnce() -> R, R>(group: &str, func: F) -> R {
     }
 }
 
+#[allow(unused)]
```

Comment:
There doesn't seem to be anything unused here. Is there a reason this is needed?

Reply:
It's only used on Windows, but when it's compiled for Linux, the Windows part is `cfg`-ed out, therefore this function is unused.